README.md

```diff
@@ -68,7 +68,7 @@ The list of actions will likely grow as we get feedback from you.
 ### Running script (Recommended)
 
 ```bash
-/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/tuist/tuist/master/install/install)"
+/usr/bin/ruby -e "$(curl -fsSL https://install.tuist.io)"
```

Comment:
I set up the `install.tuist.io` CNAME to redirect to the install script in the GCS bucket.

Reply:
Is the CNAME record updated? I can't seem to be able access this :/